source/beamline/CLS/CLSAdvancedScalerChannelDetector.h

```diff
@@ -24,119 +24,6 @@ along with Acquaman.  If not, see <http://www.gnu.org/licenses/>.
 
 #include "beamline/CLS/CLSBasicScalerChannelDetector.h"
 
-class CLSScalerChannelDetector : public AMDetector
```

Comment:
Yeah, it was my stepping stone class. It used the old CLSSIS3820Scaler.
The new class uses the new scaler so no one needed this anymore.